images/macos/provision/core/haskell.sh

```diff
@@ -4,7 +4,10 @@ curl --proto '=https' --tlsv1.2 -sSf https://get-ghcup.haskell.org | sh
 export PATH="$HOME/.ghcup/bin:$PATH"
 echo 'export PATH="$PATH:$HOME/.ghcup/bin"' >> "$HOME/.bashrc"
 
-availableVersions=$(ghcup list | grep -v "prerelease" | grep "ghc " | awk '{print $3}')
+# ghcup output looks like this "x ghc 8.8.3 base-4.13.0.0 hls-powered", need to take all the first versions only(8.8.3 in that case) and avoid pre-release ones
+availableVersions=$(ghcup list | grep -v "prerelease" | grep "ghc " | grep -Eo ' ([0-9]*\.){2}[0-9]*' | tr -d ' ')
```

Comment:
nitpick: `ghcup list -t ghc -r | awk '{print $2}'`

Reply:
I should have checked `ghcup` flags, thank you!